instance/templates/instance/ansible/vars.yml

```diff
@@ -83,5 +83,5 @@ EDXAPP_WORKER_DEFAULT_STOPWAITSECS: 1200
 COMMON_ENABLE_NEWRELIC: {% if newrelic_license_key %}true{% else %}false{% endif %}
 COMMON_ENABLE_NEWRELIC_APP: {% if newrelic_license_key %}true{% else %}false{% endif %}
 COMMON_ENVIRONMENT: opencraft
-COMMON_DEPLOYMENT: '{{ instance.sub_domain }}'
+COMMON_DEPLOYMENT: '{{ instance.internal_lms_domain }}'
```

Comment:
@mtyaka So it doesn't matter if `COMMON_DEPLOYMENT` is set to `foo.sandbox` or `foo.sandbox.example.com`? I'm not familiar with that variable, so this is a genuine question :)